packages/drupal-kit/src/fetch/defaultFetch.ts

```diff
@@ -2,28 +2,39 @@ import { ServerResponse } from 'http';
 import fetch from 'isomorphic-fetch';
 
 import addSurrogateKeyHeader from '../utils/addSurrogateKeyHeader';
-import updateMaxAge from '../utils/updateMaxAge';
+
+const defaultCacheControlValue =
+  'public, s-maxage=10, stale-while-revalidate=600';
+
 /**
  * fetch data from a JSON:API endpoint, bubbling up surrogate keys if possible
  * @param apiUrl the api url for the JSON:API endpoint
  * @param requestInit fetch initialization object
  * @param res response object
+ * @param cacheControl optional value to override cache control header, defaults to 'public, s-maxage=10, stale-while-revalidate=600'
  * @returns a promise containing the data for the JSON:API response
  */
 const defaultFetch = (
   apiUrl: RequestInfo,
   requestInit: RequestInit = {},
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  res?: ServerResponse | boolean
+  res?: ServerResponse | boolean,
```

Comment:
@CobyPear Yeah, this would have to be a Drupal State change I think. In getObject `res` has a default value of `false`. Seems like that shouldn't be necessary. Was probably past me fighting with types and admitting defeat :)